### ISSUE TITLE
[SF_ACC][DatePicker]: [UI5 Date Picker valueStateMessage tooltip error is not announced by screen reader]

### ISSUE THREAD

**Issue by cptDev6** [SAP/ui5-webcomponents#13185]
### Describe the bug

When using the date-picker component with value-state="Negative", the screen reader (JAWS) announces only "Value State Error" in all scenarios instead of the actual descriptive error message displayed in the popover/tooltip.

Error messages are not announced immediately when validation fails. Screen readers only announce the error when the field receives focus (e.g., when tabbing to the field or returning to it after navigating away).

### Isolated Example

https://ui5.github.io/webcomponents-react/v2/?path=/story/inputs-datetimepicker--default&args=primaryCalendarType:Gregorian;valueState:Negative

### Reproduction steps

1. https://ui5.github.io/webcomponents-react/v2/?path=/story/inputs-datetimepicker--default&args=primaryCalendarType:Gregorian;valueState:Negative
2.  When date value is not available or when if the provided value is inavlid then the toopltip appears
3.  When with screen reader it only announces value state errror and invalid entry for all cases like if there is no value provided or if date input provided is wrong
...

### Expected Behaviour

Screen reader should announce the actual error message from the valueStateMessage slot: "Date picker is invalid (expected MM/DD/YYYY)" which will be on tooltip.

Error messages are not announced immediately when validation fails. Screen readers only announce the error when the field receives focus (e.g., when tabbing to the field or returning to it after navigating away).

### Screenshots or Videos

https://github.com/user-attachments/assets/5544b99e-e9bb-41f0-894c-32a2d71ca7fe

<img width="1898" height="1192" alt="Image" src="https://github.com/user-attachments/assets/6ea3e1e1-8314-4b1b-8897-07d02b694b8d" />

### UI5 Web Components for React Version

2.19.1

### UI5 Web Components Version

2.19.1

### Browser

Chrome

### Operating System

windows and mac

### Relevant log output

```shell

```

### Organization

SAP Labs

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.

**Comment by MapTo0** [comment]
Hey @cptDev6,

I could not reproduce the issue on my side. Could you please recheck and enhance the example so we can have a look again?

[Example](https://ui5.github.io/webcomponents/play/#eNq9VFFv0zAQ_itenrZqrofEJJSmCNhAAgmYtPGA1JerfW28OXZkO-1K1f_O2U1QB2M8IFAqpff57rvzd3fZFtoqvB_XsTFFuS0sNFiUh-BpIZ2NaCPB1RHnrDWwWXrXWcUXzijG-cuZrY4uP1_cfL16y1JQAtKbGbDL6axAOysI2z9VjaDIYvSrQtwYTMaotC4elwoX2qI6YVvClA4pWcmsszghYJfic4Z8vnBeouLSGec5qNsuxAe-rBID__9PVzUYgckafMBIGny5ecdfZBn6oyQ14SuN69b5OCtYLzSBa61iPVW40hJ5Nk6ZtjpqMDxIMDh9Nj7ryaKOlPMamtZgJfYWqSx6menv3KkNy6UR9RzkXd--fJOSrcAfcx6gffPj6CKdnExYjXpZ0y2fn5-195M-4SNDwNH2g5AcOn3OFUTkrZZ36FnrdQN-w1PhVoHncdOmUj5ACxYD0tUD0uXVY07vg4FGS_JZgemQh0jMhH_CJUS9wr6oQFQsGJfUy47Xye8jhgDL5EPiQEDmSRQWax2oVxSRQ8VP9Q6zmWBt2y7-28w5xe-E3YtKSaTXbWS9Jo1TncmyeZlM0HZ8GygXcWfHPACp7fsBEMNSPtm6Ync6UB18Cgbk8DsgRocstVbIRmJmdZMGmc2KsTg4Dl2b4FwgbcqvsbmCB_GvSBexxrl0hFjKGQQtZxSvA-1S4AuMsh74_hh0Sd24yo0dQp4M4sbRCOpv1GJn9wwLhNh5DGKYTdEPZSb8e7ZhDzJdsdt9BxM0-JE)

**Comment by cptDev6** [comment]
@MapTo0 It's announcing value state errors for all error messages. Isn't the value state property announcement providing redundant information which user will not be aware of what value state is.

**Comment by Vonahz** [comment]
Hi @cptDev6 ,

This is by design and is connected with this issue [#539](https://github.com/UI5/webcomponents/issues/5392)
